### PR TITLE
Ioc container setup code

### LINKED ISSUE #104: Refactor: IoC container setup code
## What should be refactored?

- Create virtual folder in `Binder.Api` project for extension classes,
- Create extension classes for Service and Repository classes' registration instructions,
- Create extension class for Swagger config code,

Remember to move any additional extension classes to their respective layers, if necessary.

## Reason for refactoring:

IoC container in `Program.cs` file on `Binder.Api` backend's project is possessing all-in-one instructions. This is not following general good practices.

### PR DESCRIPTION
**Pull request by Lukash88**
## Summary

Refactor registration of IoC.

## Details

- Create virtual folder in **Binder.Api** project for extension classes,
- Create extension class for **Service** and **Repository** registration instructions,
- Create extension class for **Swagger** config code.

## References

No external references needed.

### CODE REVIEW

File: binder-web-backend/Binder.Api/Extensions/ApplicationServicesExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+using Binder.Application.Models.Interfaces;
+using Binder.Application.Services;
+using Binder.Persistence.Models.Interfaces;
+using Binder.Persistence.Repositories;
+
+namespace Binder.Api.Extensions
+{
+    public static class ApplicationServicesExtensions
```
Comment (StraykerPL) [inline binder-web-backend/Binder.Api/Extensions/ApplicationServicesExtensions.cs]: Refactor to decouple relations with Persistance and Appliacation layer.

File: binder-web-backend/Binder.Api/Extensions/SwaggerServiceExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                options.AddServer(new OpenApiServer
+                {
+                    Url = backendUrl
+                });
+            });
+
+            return services;
+        }
+
+        public static IApplicationBuilder UseSwaggerDocumentation(this IApplicationBuilder app, IWebHostEnvironment env)
```
Comment (StraykerPL) [inline binder-web-backend/Binder.Api/Extensions/SwaggerServiceExtensions.cs]: `IApplicationBuilder` and `IWebHostEnvironment` deliver same Environment value, no need for additional argument.

**Review by StraykerPL — CHANGES_REQUESTED** [review]

File: binder-web-backend/Binder.Api/Extensions/ApplicationServicesExtensions.cs (hunk 1 above)
Comment (Lukash88) [reply binder-web-backend/Binder.Api/Extensions/ApplicationServicesExtensions.cs]: Resolved a688c46.

File: binder-web-backend/Binder.Api/Extensions/SwaggerServiceExtensions.cs (hunk 2 above)
Comment (Lukash88) [reply binder-web-backend/Binder.Api/Extensions/SwaggerServiceExtensions.cs]: Resolved a688c46.

**Review by StraykerPL — APPROVED** [review]

**Pull request merged**